zerver/lib/email_mirror.py

```diff
@@ -54,15 +61,18 @@ def log_and_report(email_message, error_message, debug_info):
 
 
 def missed_message_redis_key(token):
+    # type: (text_type) -> text_type
     return 'missed_message:' + token
 
 
 def is_missed_message_address(address):
+    # type: (str) -> bool
```

Comment:
I believe `address` here is an email address. So it should be typed as `text_type`.